packages/owl-ms-language-server/package.yaml

```
@@ -0,0 +1,15 @@
+---
+name: owl-ms-language-server
+description: A language server for the OWL 2 Mancherster Syntax
+homepage: https://github.com/janekx21/owl-ms-language-server
+licenses:
+  - MIT
+languages:
+  - OWL
+categories:
+  - LSP
+source:
+  id: pkg:cargo/owl-ms-language-server@0.1.3
```

```suggestion
  id: pkg:cargo/owl-ms-language-server@0.1.3
  
schemas:
  lsp: vscode:https://raw.githubusercontent.com/janekx21/vscode-owl-ms/main/package.json
```

We should add the VS Code LSP Schema

---

> We should add the VS Code LSP Schema
@mehalter 

The VS Code LSP Schema as of now is just the example config so it doesn't add anything new as of now